main.go

```diff
@@ -320,3 +323,12 @@ func (a *Agent) Lock(passphrase []byte) error {
 func (a *Agent) Unlock(passphrase []byte) error {
 	return ErrOperationUnsupported
 }
+
+func notifyTouchRequired() {
+
+	if _, err := exec.LookPath("notify-send"); err == nil {
+		cmd := exec.Command("notify-send", "-a", "yubikey-agent", "Touch required", "Please touch your Yubikey for SSH")
```

Comment:
worth adding `-i info` to this ? 
![image](https://user-images.githubusercontent.com/1102850/83751335-924edd00-a65e-11ea-91e8-34fc3eb7c123.png) vs 
![image](https://user-images.githubusercontent.com/1102850/83751372-a09cf900-a65e-11ea-887e-481b8ecc0870.png)

Reply:
I don't think there is a standard list of icons guaranteed to be available on all distributions. If "icon" is guaranteed to be available, then why not yes.

Reply:
The standard flag is `dialog-information`  https://specifications.freedesktop.org/icon-naming-spec/latest/ar01s04.html  . thats probably a better choice.

Reply:
or `dialog-password` ?

Reply:
Dialog-password is good for me. Added it to the code.